core/ir/aarch64/instr_create_api.h

```diff
@@ -114,6 +114,9 @@
 /** Create an operand specifying LSL, the default shift type when there is no shift. */
 #define OPND_CREATE_LSL() opnd_add_flags(OPND_CREATE_INT(DR_SHIFT_LSL), DR_OPND_IS_SHIFT)
 
+/** Create an operand specifying MUL, a multiplier for SVE predicate constraints. */
```

Comment:
I think this should be a generic multiplier operand, not just for SVE.

Reply:
done